jigar/hangman/hangman.rb

```diff
@@ -0,0 +1,45 @@
+$secret_word = "aaabbbcccc"
+$word = $secret_word.split("")

```

Comment:
Rather than splitting string in array string can be used as it is for $word

Reply:
Thanks for suggestion @MayurJain-Kylas. I have done the changes